### DISCUSSION TITLE
禁用了inline_preedit（预编辑区）之后，配色不生效了。

### DISCUSSION THREAD

**Discussion by kwindva** [rime/home#1715] [Q&A] [answered]
一直用「暗堂」或者「远山」配色，刚才把weasel.yaml文件里的inline_preedit改成了false（因为嫌弃这个预编辑区，在视觉上与之前的文字行对不齐，强迫症犯了），这样打字时的拼音显示就回到了输入框上，但是现在整个输入框的配色变成了浅色，怎么改都变不了了。
包括把这个文件里的style: color_scheme: 和color_scheme_dark: 都改成了nord（远山），也都无济于事。
它现在调用的这个配色，应该是在哪儿设置出来的？

**Comment by kwindva** [comment]
![图片](https://github.com/user-attachments/assets/822474ad-05fa-43eb-b5d0-e7cccb26da87)
![图片](https://github.com/user-attachments/assets/a99632ff-388b-458a-a9ab-91d10c1af0f3)
附图如上。

**Comment by kwindva** [comment, marked as answer]
修改了程序文件夹下面，data下面的weasel.yaml，把style:color_scheme:改了就好了。。。
很奇怪为什么非得改这儿才行。。

**Comment by lotem** [comment]
別改程序。。
肯能是文件改出語法錯誤了。

**Reply by kwindva** [reply]
这样啊。。可是实在找不到问题出在哪儿。。感觉自己也没改多少东西。。
